### PR TITLE
Convert course instance LTI page to TS

### PR DESCRIPTION
**Pull request by jonatanschroeder**
This page is not expected to be maintained for long, given the imminent transition to LTI 1.3. It was converted only to remove some dependencies on partials and similar components. In particular, some common modernization efforts were not performed here, such as:

* convert res.locals values to parameters to the html.ts file;
* convert the query to a Zod validated query;
* create the db-types schema and type for lti_links;
* move date formatting from query to TS;
* convert the JS client side code to an asset.

I have limited ability to test the LTI links portion of the page. Someone with access to staging or a local LTI 1.1 installation may need to test it.

### CODE REVIEW

**Comment by jonatanschroeder** [comment]
@trombonekenny pinging you as requested.

**Comment by trombonekenny** [comment]
@trombonekenny Hey, review this.

File: apps/prairielearn/src/pages/instructorInstanceAdminLti/instructorInstanceAdminLti.html.ts [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+                        class="far fa-copy"
+                        title="Copy to clipboard"
+                        onClick="copyToClipboard($(this).prev());$(this).fadeOut({queue: true});$(this).fadeIn({queue:true});"
+                      ></i>
+                    </td>
+                    <td>${tc.created}</td>
+                    <td>
+                      ${tc.deleted ||
+                      html`
+                        <a
```
Comment (nwalters512) [inline apps/prairielearn/src/pages/instructorInstanceAdminLti/instructorInstanceAdminLti.html.ts:168]: I'd say "just use a `<button>`" if there wasn't the caveat about you skipping some common modernization things.

**Review by nwalters512 — APPROVED** [review]
I tested this with my local Canvas installation and everything works as expected. Approving given the fact that we aren't _actually_ modernizing much here.

File: apps/prairielearn/src/pages/instructorInstanceAdminLti/instructorInstanceAdminLti.html.ts (hunk 1 above)
Comment (jonatanschroeder) [reply apps/prairielearn/src/pages/instructorInstanceAdminLti/instructorInstanceAdminLti.html.ts:168]: Yeah, in this case I'll rely on "Just Works :tm:" and merge this without changing it so we can move on with partials.

**Pull request merged**